src/renderer/features/proxies/workers/proxy-worker.ts

```diff
@@ -191,22 +190,22 @@ async function getProxies({
     console.log(e);
   }
 
-  const proxiesToRemove = proxies.filter((p) => existingProxies.every((ep) => isEqual(p, ep)));
-
-  const proxiedAccountsToRemove = Object.values(proxiedAccounts)
-    .filter(proxyWorkerUtils.isProxiedAccount)
-    .filter(
-      (p) =>
-        !existingProxiedAccounts.some(
-          (ep) =>
-            ep.accountId === p.accountId &&
-            ep.chainId === p.chainId &&
-            ep.proxyAccountId === p.proxyAccountId &&
-            ep.proxyVariant === p.proxyVariant &&
-            ep.delay === p.delay &&
-            ep.proxyType === p.proxyType,
-        ),
-    );
+  const proxiesToRemove = proxies.filter((p) => !existingProxies.some((ep) => proxyWorkerUtils.isSameProxy(p, ep)));
+  console.log(`proxy-worker ${api.genesisHash}: proxies👩‍💼 accounts to remove: `, proxiesToRemove);
+
+  const proxiedAccountsToRemove = Object.values(proxiedAccounts).filter(
```

Comment:
Please add `{ }` instead of pure arrow functions, because it's pretty hard to read